test/int.jl

```diff
@@ -142,3 +142,27 @@ end
 @test widemul(false, false) == false
 @test widemul(false, 3) == 0
 @test widemul(3, true) == widemul(true, 3) == 3
+@test widemul(Int64(2), Int64(3)) == Int128(6)
```

Comment:
If you use `==` for comparison, then you can leave out the explicit conversion to `Int128`. On the other hand, it does make sense to check the return type as well, in which case you'll want t use `===` for the comparison instead of `==`.